external/WICED/bcm4390x/include/wiced_utilities.h

```diff
@@ -0,0 +1,112 @@
+/****************************************************************************
+ *
+ * Copyright 2018, Cypress Semiconductor Corporation or a subsidiary of
+ * Cypress Semiconductor Corporation. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#pragma once
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/******************************************************
+ * @cond                       Macros
+ ******************************************************/
+
+#ifdef LINT
+/* Lint does not know about inline functions */
+extern uint16_t htobe16(uint16_t v);
+extern uint32_t htobe32(uint32_t v);
+
+#else							/* ifdef LINT */
+
+#endif							/* ifdef LINT */
+
+#ifndef MIN
+#define MIN(x,y)  ((x) < (y) ? (x) : (y))
+#endif							/* ifndef MIN */
+
+#ifndef MAX
+#define MAX(x,y)  ((x) > (y) ? (x) : (y))
+#endif							/* ifndef MAX */
+
+#ifndef ROUND_UP
+#define ROUND_UP(x,y)    ((x) % (y) ? (x) + (y)-((x)%(y)) : (x))
```

Comment:
Could you check all of comma for coding rule?
We must have a space after comma.
```
MIN(x, y)
```

same as ```-```, ```%``` and ```==```.
NOTE!! not allowed a spcae after type casting.

Reply:
it will be changed as your guidance.
BR, HJ Lee.